### DISCUSSION TITLE
allDub/allSub

### DISCUSSION THREAD

**Discussion by drknss89** [anidl/multi-downloader-nx#138] [Q&A] [answered]
I am having trouble getting allSub and allDubs to work.  Still defaults to english

.\anidl.exe --service "funi" -s 1007443 -e 1 -q 0 -allDub --allSubs  --fileName "${showTitle} - S${season}E${episode} - ${title}

Where am i going wrong?

**Comment by DinoVikingo** [comment]
Hi!
Mine says "track not selected":(

**Comment by izu-co** [comment, marked as answer]
If you think you've spotted an error please open up an issue instead of a discussion.

Thank you :)
